mult.py

```diff
@@ -0,0 +1,32 @@
+def least_common_multiple(big_num, small_num):
+    count = 1
+    while True:
+        if (big_num * count)%small_num == 0:
+            return big_num * count
+        else:
+            count+=1
+
+
+# read numbers from user
+while True:
+    try:
+        a = int(input('enter first number\n'))
+    except ValueError:
+        print('enter valid input')
+        continue
+    else:
+        break
+
+while True:
+    try:
+        b = int(input('enter second number\n'))
+    except ValueError:
+        print('enter valid input')
+        continue
+    else:
+        break
+
+big_num = max(a, b)

```

Comment:
Why not figure big and small inside the `least_common_multiple` function?
It's not really the user's concern..

Reply:
Done